### ISSUE TITLE
New UI is not supported?

### ISSUE THREAD

**Issue by shinich39** [Ace4896/project-sekai-thumbnail-extractor#1]
New ui coming soon.

**Comment by Ace4896** [comment]
You're right, this extractor app (and my [thumbnail matching](https://github.com/Ace4896/project-sekai-thumbnail-matcher) app) only supports the old UI. When I originally wrote this a year ago, JP had just switched to new UI, so it really has been that long 😅

I had a quick look at the new UI's character screen - the approach I used might be viable, but it seems some of the image thresholding values need tweaking because of the translucent backgrounds.
